exchanges/btse/btse_websocket.go

```diff
@@ -191,28 +166,34 @@ func (b *BTSE) GenerateDefaultSubscriptions() {
 
 // Subscribe sends a websocket message to receive data from the channel
 func (b *BTSE) Subscribe(channelToSubscribe wshandler.WebsocketChannelSubscription) error {
-	subscribe := websocketSubscribe{
-		Type: "subscribe",
-		Channels: []websocketChannel{
-			{
-				Name:       channelToSubscribe.Channel,
-				ProductIDs: []string{channelToSubscribe.Currency.String()},
-			},
-		},
-	}
-	return b.WebsocketConn.SendMessage(subscribe)
+	var sub wsSub
+	sub.Operation = "subscribe"
+	sub.Arguments = []string{channelToSubscribe.Channel}
+	return b.WebsocketConn.SendMessage(sub)
 }
 
 // Unsubscribe sends a websocket message to stop receiving data from the channel
 func (b *BTSE) Unsubscribe(channelToSubscribe wshandler.WebsocketChannelSubscription) error {
-	subscribe := websocketSubscribe{
-		Type: "unsubscribe",
-		Channels: []websocketChannel{
-			{
-				Name:       channelToSubscribe.Channel,
-				ProductIDs: []string{channelToSubscribe.Currency.String()},
-			},
-		},
+	var unSub wsSub
+	unSub.Operation = "unsubscribe"
+	unSub.Arguments = []string{channelToSubscribe.Channel}
+	return b.WebsocketConn.SendMessage(unSub)
+}
+
+// Pinger pings
+func (b *BTSE) Pinger() {
+	count := 57 * time.Second
+	ticker := time.NewTicker(count)
+
+	for {
+		select {
+		case <-b.Websocket.ShutdownC:
+			ticker.Stop()
+			return
+
+		case <-ticker.C:
+			log.Debug("Sending ping")
```

Comment:
wrap this in a b.Verbose comparative.

Reply:
I just deleted it. we were using it for testing forgot to remove it afterwards